api/event_test.go

```diff
@@ -82,23 +82,23 @@ func (s *EventSuite) generateEventCustomData(appName string) *app.DeployOptions
 	return &app.DeployOptions{
 		App: &app.App{
 			Name: appName,
-			Env: map[string]bind.EnvVar{
+			Env: map[string]appTypes.EnvVar{
 				"MY_PASSWORD": {
 					Name:   "MY_PASSWORD",
 					Public: false,
 					Value:  "password123",
 				},
 			},
-			ServiceEnvs: []bind.ServiceEnvVar{
+			ServiceEnvs: []appTypes.ServiceEnvVar{
```

Comment:
I think we could take the opportunity to move EnvVar and ServiceEnvVar to a more generic package like `types/`